browser rendering limits (#19566)
cloudflare/cloudflare-docs: src/content/changelogs-next/2025-01-30-browser-rendering-more-instances.mdx

---
title: More concurrent Br
description: Browser Rendering now supports more concurrent browser sessions.
products:
  - workers
  - browser-rendering
date: 2025-01-30T13:00:00Z
---

import { Render, PackageManagers, TypeScriptExample } from "~/components"

> [Browser Rendering](/browser-rendering/) now supports 10 concurrent browser instances per account, up from the previous limit of 2. This allows you to launch more browser tasks from [Cloudflare Workers](/workers).

Author: ```suggestion
[Browser Rendering](/browser-rendering/) now supports 10 concurrent browser instances per account _and_ 10 new instances per minute, up from the previous limits of 2.

This allows you to launch more browser tasks from [Cloudflare Workers](/workers).
```

> [Browser Rendering](/browser-rendering/) now supports 10 concurrent browser instances per account, up from the previous limit of 2. This allows you to launch more browser tasks from your Workers. We'll continue to work on increasing the number of concurrent browsers as the Browser Rendering beta progresses.

Reviewer: ```suggestion
[Browser Rendering](/browser-rendering/) now supports 10 concurrent browser instances per account, up from the previous limit of 2. This allows you to launch more browser tasks from [Cloudflare Workers](/workers). We'll continue to work on increasing the number of concurrent browsers as the Browser Rendering beta progresses.
```

> [Browser Rendering](/browser-rendering/) now supports 10 concurrent browser instances per account, up from the previous limit of 2. This allows you to launch more browser tasks from [Cloudflare Workers](/workers). We'll continue to work on increasing the number of concurrent browsers as the Browser Rendering beta progresses.

Author: ```suggestion
[Browser Rendering](/browser-rendering/) now supports 10 concurrent browser instances per account, up from the previous limit of 2. This allows you to launch more browser tasks from [Cloudflare Workers](/workers).
```
